SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -42,87 +42,107 @@ In this step, you're going to build the Unreal GDK's dependencies.
 1. In the same directory, open **ThirdPersonShooter.sln** with Visual Studio.
 1. In the Solution Explorer window, right-click on **ThirdPersonShooter** and select **Build**.
 1. Open **ThirdPersonShooter.uproject** in the Unreal Editor.
-2. In the SpatialOS GDK toolbar, click [`Schema` (SpatialOS documentation)]({{urlRoot}}/content/glossary#schema) to generate schema and then [`Snapshot` (SpatialOS documentation)]({{urlRoot}}/content/glossary#snapshot) to generate a snapshot.
+1. In the [GDK toolbar (SpatialOS documentation)]({{urlRoot}}/content/toolbars), click [`Schema` (SpatialOS documentation)]({{urlRoot}}/content/glossary#schema) to generate schema and then [`Snapshot` (SpatialOS documentation)]({{urlRoot}}/content/glossary#snapshot) to generate a snapshot.
 
 ### Deploy the project locally
 
 In this section you’ll run a [local deployment](https://docs.improbable.io/reference/latest/shared/glossary#local-deployment) of the project. As the name suggests, local deployments run on your development machine (you will [cloud deploy](https://docs.improbable.io/reference/latest/shared/glossary#cloud-deployment) later in this tutorial).
 
-1. In a text editor, open `UnrealGDKThirdPersonShooter\spatial\default_launch.json` and set `rows` to `2`. This instructs SpatialOS that we will be launching two server-workers.
+1. In a text editor, open `UnrealGDKThirdPersonShooter\spatial\default_launch.json` and set `rows` to `2`. This instructs SpatialOS that you will be launching two server-workers.
```

Comment:
Provide the section here , maybe something like "under the load_balancing/layer_configurations/rectangle_grid." This is what the Json look like:
"load_balancing": {
    "layer_configurations": [
      {
          "layer": "UnrealWorker",
          "rectangle_grid": {